recipe/bld.bat

```diff
@@ -1,9 +1,34 @@
 @echo on
-setlocal enabledelayedexpansion
-
-:: copy %SRC_DIR%/base somewhere to be able to hard-reset state in build-output.bat
-mkdir .\backup
-robocopy base backup /E >nul
-:: for whatever reason, robocopy returns 1 in case of success, see
-:: https://learn.microsoft.com/en-us/troubleshoot/windows-server/backup-and-storage/return-codes-used-robocopy-utility
-if %ERRORLEVEL% neq 1 exit 1
+
+mkdir builddir
+
+:: check if clang-cl is on path as required
+clang-cl.exe --version
+if %ERRORLEVEL% neq 0 exit 1
+
+:: set compilers to clang-cl
+set "CC=clang-cl"
+set "CXX=clang-cl"
+
+:: flang 17 still uses "temporary" name
+set "FC=flang-new"
+
+:: set up clang-cl correctly, see
+:: https://github.com/conda-forge/clang-win-activation-feedstock/blob/main/recipe/activate-clang_win-64.sh
+set "CPPFLAGS=-D_CRT_SECURE_NO_WARNINGS -D_MT -D_DLL --target=x86_64-pc-windows-msvc -Xclang --dependent-lib=msvcrt -fuse-ld=lld"
+set "FFLAGS=-D_CRT_SECURE_NO_WARNINGS -D_MT -D_DLL --target=x86_64-pc-windows-msvc -nostdlib"
+set "LDFLAGS=--target=x86_64-pc-windows-msvc -nostdlib -Xclang --dependent-lib=msvcrt -fuse-ld=lld"
+set "LDFLAGS=%LDFLAGS% -Wl,-defaultlib:%LIBRARY_PREFIX%/lib/clang/17.0.0/lib/windows/clang_rt.builtins-x86_64.lib"
```

Comment:
Use `clang -dumpversion` instead of hardcoding?

Reply:
Yeah, that was a leftover from trying to follow your [request](https://github.com/conda-forge/scipy-feedstock/pull/246#issuecomment-1732692927) of matching the flags from the clang-win activation.

However, it's possible that compiler-rt is not actually necessary? I guess we'd have to try, because as a static lib it doesn't show up in the link check.